external/test-runtime/project.json

```diff
@@ -3,10 +3,14 @@
     "xunit.console.netcore": "1.0.3-prerelease-00921-01",
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.1-prerelease-01001-04",
     "Microsoft.xunit.netcore.extensions": "1.0.1-prerelease-01001-04",
-    "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0040",
-    "Microsoft.DotNet.xunit.performance.analysis.cli": "1.0.0-alpha-build0040",
-    "Microsoft.DotNet.xunit.performance.runner.cli": "1.0.0-alpha-build0040",
-    "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0040",
+    "xunit.performance.api": "1.0.0-alpha-build0047",
+    "xunit.performance.core": "1.0.0-alpha-build0047",
+    "xunit.performance.execution": "1.0.0-alpha-build0047",
+	"xunit.performance.metrics": "1.0.0-alpha-build0047",
+	"Microsoft.Diagnostics.Tracing.TraceEvent": "1.0.0-alpha-experimental",
+	"Microsoft.3rdpartytools.MarkdownLog": "0.10.0-alpha-experimental",
```

Comment:
Is MarkdownLog on MyGet yet, or are you pulling this from a local feed via a Nuget.config?